rc/tools/lint.kak

```diff
@@ -89,12 +89,14 @@ define-command \
                     '
                         BEGIN { OFS=":"; FS=":" }
 
-                        /:[1-9][0-9]*:[1-9][0-9]*:/ {
+                        /^[^:]+:([1-9][0-9]*:){1,2} / {
                             $1 = ENVIRON["kak_bufname"]
-                            if ( $2 == 1 ) {
+                            $2 += line_offset
+                            if ( $3 ~ /^ / ) {
+                                $2 = $2 FS
+                            } else if ( $2 == 1 + line_offset ) {
```

Comment:
I dont like adding line offset then comparing with line_offset + 1, it seems convoluted, surely we can just store that information in a boolean like variable if we need to check it before modifying $2.